contract_api/consumers/service_event_consumer.py

```diff
@@ -112,6 +112,60 @@ def _get_new_assets_url(self, org_id, service_id, new_ipfs_data):
                                                                  service_id)
         return assets_url_mapping
 
+    def create_service_media(self,org_id,service_id,service_media,service_row_id):
+        count = 0;
+        if len(service_media)>0:
+            #clear the existing values from db
+            self._service_repository.delete_service_media(org_id=org_id,service_id=service_id)
+
+            #for video -> if ipfs_url store in s3 and update url else store url
+            for service_media_item in service_media:
+
+                url = service_media_item.get("url",{})
+                if service_media_item.get("file_type",{}) == "video" and ("http" in url or "https" in url):
```

Comment:
Why is this check specific to video files??

Reply:
this is corrected